manifests/kustomize/README.md

```diff
@@ -48,7 +48,22 @@ kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80
 ```
 and open http://localhost:8080/
 
+## Configure Metadata Server
 
+The Pipeline metadata server deployment requires mysql credentials to be
+configured as Kubernetes Secrets. Following command can be used to create the
+required secret object.
+
+```
+kubectl create secret generic mysql-credential --namespace <your-namespace> --from-literal=username=<your-mysql-username> --from-literal=password=<your-mysql-password>
```

Comment:
What's the default username and password as used by APIServer? Is it still `root` and empty password? If so, let's mention that here, and tell the user to create it with these defaults.

Should we just create this secret as part of the deployment script? This should be the default, and we can call out that this is what needs to be changed by the user if re-using an existing DB (e.g. CloudSQL).

Reply:
The default is still "root" and empty password. Will update the doc to reflect this.

By deployment script you mean for Marketplace? I am sure if lite deployment has one?

Reply:
Sorry, I mean this pipeline-lite deployment, not marketplace.

Essentially, since the default user and pass is also used by APIServer, so why not create a secret that uses the same settings? 

/cc @IronPan

Reply:
I like the idea, essential create a Secret object via yaml files and then use that across other components. Let me update the PR with that change.